pages/attributes.vue

```diff
@@ -0,0 +1,342 @@
+<template>
+  <v-layout column>
+    <h1 class="display-4 ma-5">Attributes</h1>
+
+    <p :class="paragraph">
+      In this guide, you will learn fundamental elements to understand and 
+      manipulate attributes in OpenGeode.
+    </p>
+
+    <h2 :class="section">What are attributes?</h2>
+
+    <p
+      :class="paragraph"
+    >Attributes stand for properties you can add on OpenGeode objects, typically meshes. 
+    These properties are not directly stored on meshes, so you can imagine add Attributes
+    on other objects. </p>
+
+    <p
+      :class="paragraph"
+    >Attributes are managed by an 
+    <span v-html="attributeManager" />.
+    Meshes have manager defined on their vertices, edges, facets, and so on.
+    Updating the mesh (e.g. by adding or removing vertices) automatically updates attached
+    <span v-html="attributeManager" />. </p>
+
+    <p
+      :class="paragraph"
+    >But you can create an
+    <span v-html="attributeManager" />
+    that is not attached to a mesh.</p>
+    
+    <h2 :class="section">Create an Attribute</h2>
+
+    <p
+      :class="paragraph"
+    >First of all, let's get an  
+    <span v-html="attributeManager" />
+    by the dedicated mesh methods or by creating it:</p>
+
+    <!-- eslint-disable -->
+    <code class="cpp primary--text">
+      AttributeManager manager;
+      manager.resize( 10 );
+      auto attribute_ptr = manager.find_or_create< VariableAttribute, double >(
+        "first_attribute", 12.3 );
+      manager.resize( 6 );
+    </code>
+    <!-- eslint-enable -->
+
+    <p
+      :class="paragraph"
+    >The first line of the above code creates a default  
```

Comment:
Fifi: Is it better with line numbers written in the code?

Reply:
Yes you are right, we will improve that.